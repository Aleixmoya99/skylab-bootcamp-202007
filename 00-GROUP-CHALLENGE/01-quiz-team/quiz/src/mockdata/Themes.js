/* quizteam - skylab bootcamp 202007 */
/* mockdata */
'use strict';

export default THEMES_LIST = ['General Knowledge', 'Entertainment: Books', 'Entertainment: Film', 'Entertainment: Music',
 'Entertainment: Musicals & Theatre', 'Entertainment: Television', 'Entertainment: Video Games',
 'Entertainment: Board Games', 'Science & Nature', 'Science: Computers', 'Science: Mathematics',
 'Mythology', 'Sports', 'Geography', 'History', 'Politics', 'Art', 'Celebrities', 'Animals', 'Vehicles',
 'Entertainment: Comics', 'Science: Gadgets', 'Entertainment: Japanese Anime & Manga', 'Entertainment: Cartoon',
 'Animations'];		
