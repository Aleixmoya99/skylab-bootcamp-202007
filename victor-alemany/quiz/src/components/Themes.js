import React from "react";

function Themes(props){
    return;
}

export default Themes;