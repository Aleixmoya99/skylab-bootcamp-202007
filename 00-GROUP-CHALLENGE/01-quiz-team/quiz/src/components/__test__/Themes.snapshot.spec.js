import React from 'react';
import renderer from 'react-test-renderer';
import Themes from './../Themes';

const tree = renderer.create(<Themes />).toJSON();

describe('Themes snapshot', () => {
    test('should match', () => {
        expect(tree).toMatchSnapshot();
    })
})