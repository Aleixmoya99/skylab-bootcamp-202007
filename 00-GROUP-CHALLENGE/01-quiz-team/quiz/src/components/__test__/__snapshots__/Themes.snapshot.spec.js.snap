// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Themes snapshot should match 1`] = `
<div
  className="theme-container"
>
  <h4>
    Sports
  </h4>
</div>
`;
